api/user/__init__.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint
+from .authors import authors_bp
+from .followers import followers_bp
+from .posts import posts_bp
+
+user_bp = Blueprint("users", __name__)
+user_bp.register_blueprint(authors_bp)
+user_bp.register_blueprint(followers_bp)
```

Comment:
For the followers instead of registering it against user [will check later if we need a user_bp? I am not sure.
It makes more sense it register it as:
```py
Author_BP = Blueprint("authors'", __name__, url_prefix="/authors"
Followers_BP = Blueprint("authors'", __name__, url_prefix="/<string:author_id>/followers"
Author_BP.register_blueprint(Followers_BP)
```

This will help us not have to trace long endpoints and will be easier to trace and less mistakes?

Reply:
```py
Blueprint("authors'", __name__, url_prefix="/<string:author_id>/followers")
```
I don't think parameters in the url prefix are supported

Reply:
Ok, they are supported, but I'm a little hestitant to go this far, because it abstracts away where the author ID is coming from. Anyone reading the route alone will have to have prior knowledge of blueprints to see what's going on here

Reply:
> Ok, they are supported, but I'm a little hestitant to go this far, because it abstracts away where the author ID is coming from. Anyone reading the route alone will have to have prior knowledge of blueprints to see what's going on here

Yes, that's a fair point! I wonder if there's another way we can take advantage of these? Instead of having to have two different ways of setting things up.